### PR TITLE
PR Review

### PR DESCRIPTION
**Pull request by oscarmild**
@fromgopi

### CODE REVIEW

File: .env.sample [hunk 1]
```diff
@@ -1,3 +1,7 @@
+NAME=framework
```
Comment (fromgopi) [inline .env.sample:1]: @oscarmild Change the name from #Framework to #cookbook-api

File: Pipfile [hunk 2]
```diff
@@ -3,13 +3,13 @@ url = "https://pypi.org/simple"
 verify_ssl = true
 name = "pypi"
 
-[packages]
+[packages]
```
Comment (fromgopi) [inline Pipfile:6]: @oscarmild Remove spaces

File: Pipfile [hunk 3]
```diff
@@ -3,13 +3,13 @@ url = "https://pypi.org/simple"
...
 
-[packages]
+[packages] 
 flask = "2.0.2"
 flask-restful = "*"
 python-dotenv = "*"
 flask-mongoengine = "*"
 mongoengine = "*"
-marshmallow = "*"
+marshmallow = "*"
```
Comment (fromgopi) [inline Pipfile:12]: @oscarmild use exact versions in the pipenv file.

**Review by fromgopi — APPROVED** [review]
Approved.

File: .env.sample (hunk 1 above)
Comment (oscarmild) [reply .env.sample:1]: Yes I will change.

**Pull request merged**